tests/playwright/src/model/pages/run-image-page.ts

```diff
@@ -81,7 +93,15 @@ export class RunImagePage extends BasePage {
       throw Error(`Start Button not enabled: ${errMessage}`);
     }
     await this.startContainerButton.click();
-    return new ContainersPage(this.page);
+    // After clicking on the button there seems to be for possible outcomes
+    // 1. Opening particular container's details page with tty tab opened
+    // 2. Opening Containers page with new container on it
+    // 3. staying on the run image page with an error
+    // 4. Starting a container without entrpoint or command creates a container, but it stays on Run Image Page without error
+    if ((await this.errorAlert.count()) > 0 && (await this.name.isVisible())) {
```

Comment:
just for readability I think it will be easier with intermediate variables

Reply:
Yes. I have changed the code a bit, added a wait for page to switch (or not) just to make sure to get right error in the line.